docs/source/traits_user_manual/notification.rst

```diff
@@ -453,21 +453,24 @@ not belong to the same object.
 
 .. _decorator-semantics:
 
-Decorator Semantics
-:::::::::::::::::::
-
+Registration time
```

Comment:
Not sure the title needs to change here, but if you do, you probably should update the id of the section on line 454, and also fix any references to the id elsewhere in the document.

Reply:
Agreed the title can be changed back. Will do that. Thanks!